codegen.yml

```
@@ -2,6 +2,8 @@ generates:
   ./fe/lib/types/seek.graphql.ts:
     schema: ${SEEK_SCHEMA_PATH:https://graphql.seek.com/graphql}
     documents:
+      - fe/lib/components/JobCategorySelect/queries.ts
+      - fe/lib/components/JobCategorySuggest/queries.ts
       - fe/lib/components/LocationSuggest/queries.ts
```

Should we wildcard this to e.g.

```suggestion
      - fe/lib/components/**/queries.ts
```